angular/README.md

```diff
@@ -1,15 +1,62 @@
-# Angular and Wootric Tutorial
+# `angular` — the wootric configured project for Angular apps
 
-![](/angular/angular-wootric-screenshot.png)
+This project is an application skeleton for a typical [Wootric](https://www.wootric.com/) setup web app. You can use it
+to quickly bootstrap your wootric setup angular webapp projects.
 
-## Requirements
+The app doesn't do much, just shows how to setup the wootric snippet for Anuglar projects.
 
-- [Angular CLI](https://cli.angular.io)
+## Getting Started
 
-## Instructions
+To get you started simply follow the instructions below:
 
-- `git clone git@github.com:Wootric/spa-examples.git`
-- `cd spa-examples/angular`
-- Open `src/app/wootric-survey.directive.ts`.
-- Replace the account token on line 14 with your account token.
-- Run the command `ng serve` to start the development server and see your survey.
+### Prerequisites
+
+You need git to clone the [spa-examples](https://github.com/Wootric/spa-examples) repository. You can get git from [here](https://git-scm.com/).
+
+We also use a number of Node.js tools to install `angular-cli`. You must have Node.js and its package manager (npm) installed. You can get them from [here](https://nodejs.org/). Then install Angular CLI globally with command `npm install -g @angular/cli`
+
+### Clone `spa-examples`
+
+Clone the `spa-examples` repository using git:
+
+```
+git clone https://github.com/Wootric/spa-examples.git
+cd spa-exaples
+cd angular
+npm install
+````
+
+### Setup Wootric (using angular directive)
+1. Load Wootric JS SDK and setup wootric: Open `src/app/wootric.directive.ts` and add below script to the file where you load all other 3rd party scripts:
+
+```
+//Add below code to WootricDirective constructor function
+window['wootric_no_surveyed_cookie'] = true; // Shows survey immediately for testing purposes.  TODO: Comment out for production.
+window['wootric_survey_immediately'] = true; // Bypass cookie based throttle for testing purposes.  TODO: Comment out for production.
+window['wootricSettings'] = {
+  email: 'nps@example.com', // TODO: Required to uniquely identify a user. Email is recommended but this can be any unique identifier.
+  created_at: 1234567890, // TODO: The current logged in user's sign-up date as a 10 digit Unix timestamp.
+  account_token: 'NPS-xxxxxxxx' // This is your unique account token.
+};
+>> Note: Update `account_token` with your account token.
+
+//Add below code to WootricDirective ngAfterViewInit function
+var script = document.createElement('script');
+
+script.type = 'text/javascript';
+script.src = 'https://cdn.wootric.com/wootric-sdk.js';
+script.async = true;
+script.onload = function() {
+  window['WootricSurvey'].run();
+};
+
+this.elementRef.nativeElement.appendChild(script);
+```
```

Comment:
This code is already inside of `src/app/wootric.directive.ts`. Maybe you can just tell them to update the account token on line 14.